docs/UserGuide.md

```diff
@@ -68,6 +68,50 @@ not be so obvious!
 1. Refer to the [Features](#features) below for details of each command.
 
 --------------------------------------------------------------------------------------------------------------------
+#### 3.2.2. General Symbols and Syntax 
+
+The following table gives you general symbols and syntax used in this user guide.
+
+| Syntax |  What it means |
+|----------|-------------|
+| `command` |  The grey block specifies the execuable command which can be entered in the command box |
```

Comment:
I think that you are missing the "italics" format for the terms related to tCheck's GUI